### PR TITLE
Add more description for firmware update.

### PR DESCRIPTION
**Pull request by jyao1**

### CODE REVIEW

File: src/08-attestation.adoc [hunk 1]
```diff
@@ -153,6 +158,12 @@ Pre-update attestation gives a chance to unbind TDI before runtime update,
 while post-update reattestation can only unbind TDI after runtime update. That
 can remove the gap between update time and attestation time.
 
+> **Note**
+> In this mode, the device will perform updated anyway, no matter the TSM/TVM
+accept the new firmware update or not. If TSM or TVM cannot accept the new
```
Comment (sameo) [inline src/08-attestation.adoc:163]:
```suggestion
> In this mode, the device will be updated, regardless of the TSM or the TVM accepting the firmware
update or not. If TSM or TVM cannot accept the new
```

File: src/08-attestation.adoc [hunk 2]
```diff
@@ -153,6 +158,12 @@ Pre-update attestation gives a chance to unbind TDI before runtime update,
...
+firmware, the TSM shall terminate the SPDM session with `END_SESSION`,
```
Comment (sameo) [inline src/08-attestation.adoc:164]: Either the TSM terminates the SPDM session or the TVM unbinds from the TDI?

File: src/08-attestation.adoc [hunk 3]
```diff
@@ -153,6 +158,12 @@ Pre-update attestation gives a chance to unbind TDI before runtime update,
...
+but not acknowledge the event with `EVENT_ACK`.
+
```
Comment (sameo) [inline src/08-attestation.adoc:166]: We have not defined `EVENT_ACK` in the previous section. Should we add a few line to describe it there?

File: src/08-attestation.adoc (hunk 2 above)
Comment (jyao1) [reply src/08-attestation.adoc:164]: Yes, that is better way to keep SPDM session. Just unbind TDI should be enough.

File: src/08-attestation.adoc (hunk 3 above)
Comment (jyao1) [reply src/08-attestation.adoc:166]: We can say "SPDM event", and SPDM message `EVENT_ACK`

**Review by sameo — APPROVED** [review]

**Pull request merged**